sagemaker-pipelines/tabular/local-mode/sagemaker-pipelines-local-mode.ipynb

```diff
@@ -0,0 +1,1132 @@
+{
```

Comment:
<p>Are there any resources that should be deleted that would otherwise incur ongoing costs?</p><p> </p><p>Add a Conclusion section summarizing what the customer accomplished and any recommended next steps.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3549/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Reply:
No resources are created in this notebook. Added conclusion.